certified-connectors/Docurain/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API \u30ad\u30fc",
+          "description": "\u3053\u306e API \u306e API \u30ad\u30fc",
```

Comment:
@geckoR42 Please remove unicodes from the apiProperties.json file. If these unicodes are belongs to non-English characters, please replace them with English characters.

Reply:
@sriyen-msft Fixed "displayName" and "description"